test/vanilla/src/main/java/fixtures/bodybyte/implementation/BytesImpl.java

```diff
@@ -296,6 +312,11 @@ public Completable putNonAsciiAsync(@NonNull byte[] byteBody) {
      */
     public Maybe<byte[]> getInvalidAsync() {
         return getInvalidWithRestResponseAsync()
-            .flatMapMaybe((BodyResponse<byte[]> res) -> res.body() == null ? Maybe.empty() : Maybe.just(res.body()));
+            .flatMapMaybe(new Function<BodyResponse<byte[]>, Maybe<byte[]>>() {
+                public Maybe<byte[]> apply(BodyResponse<byte[]> res) {
+                    return res.body() == null ? Maybe.<byte[]>empty() : Maybe.just(res.body());
+                }
+            }
+            );
```

Comment:
Hmm... I'm not really a fan of the right parenthesis being on the next line after the right curly-bracket. Since this for a one-off thing I think it's fine, but if you have some extra time you might look into figuring out a way to fix this. This is the sort of thing that makes people shudder when they think of "generated code".

Reply:
I think I figured out an easy fix for this in a later comment.